tests/python/test_python_frontend.py

```diff
@@ -448,7 +448,7 @@ def nvfuser_fusion(
             eps=1e-12,
             keepDim=True,
         )
-        nvf_out, _ = self.exec_nvfuser(fusion_func, inputs)
+        nvf_out, _ = self.exec_nvfuser(fusion_func, inputs, is_clonable=True)
```

Comment:
qq: this function doesn't use welford, is this just catching something missed out earlier?